mod/contacts.php

```diff
@@ -635,15 +636,15 @@ function contacts_content(App $a, $update = 0)
 			'$follow_text' => $follow_text,
 			'$profile_select' => $profile_select,
 			'$contact_id' => $contact['id'],
-			'$block_text' => ($contact['blocked'] ? L10n::t('Unblock') : L10n::t('Block')),
-			'$ignore_text' => ($contact['readonly'] ? L10n::t('Unignore') : L10n::t('Ignore')),
-			'$insecure' => (in_array($contact['network'], [Protocol::ACTIVITYPUB, Protocol::DFRN, Protocol::MAIL, Protocol::DIASPORA]) ? '' : $insecure),
+			'$block_text' => (($contact['blocked']) ? L10n::t('Unblock') : L10n::t('Block') ),
```

Comment:
I didn't do these changes.

Reply:
Indeed, you're reverting recent changes by Michael. This means your `issue_login_redirect` needs to be rebased onto `develop`. Here's how in your Friendica folder: 
```
git fetch --all
git rebase friendica/develop issue_login_redirect
git push --force
```

Reply:
after rebase it just sad everything is uptodate.. I keep trying around